langserver/internal/refs/refs_test.go

```diff
@@ -71,49 +71,49 @@ func TestParseFile(t *testing.T) {
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "Client Transport"}, Position: pos("testdata/convoluted.go:15:4 (offset 169)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "RoundTripper RoundTrip"}, Position: pos("testdata/convoluted.go:19:25 (offset 301)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "Client Transport"}, Position: pos("testdata/convoluted.go:19:15 (offset 291)")},
 			},
 		},
 		{
 			Filename: "testdata/defs.go",
 			Want: []*Ref{
-				&Ref{Def: Def{ImportPath: "net/http", Path: ""}, Position: pos("testdata/defs.go:3:8 (offset 21)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "Client"}, Position: pos("testdata/defs.go:6:10 (offset 78)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "RoundTripper"}, Position: pos("testdata/defs.go:7:9 (offset 119)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "Client"}, Position: pos("testdata/defs.go:10:21 (offset 182)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "Client"}, Position: pos("testdata/defs.go:12:12 (offset 246)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "RoundTripper"}, Position: pos("testdata/defs.go:13:11 (offset 295)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "Client"}, Position: pos("testdata/defs.go:20:13 (offset 392)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: ""}, Position: pos("testdata/defs.go:3:8 (offset 21)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "Client"}, Position: pos("testdata/defs.go:6:10 (offset 78)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "RoundTripper"}, Position: pos("testdata/defs.go:7:9 (offset 119)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "Client"}, Position: pos("testdata/defs.go:10:21 (offset 182)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "Client"}, Position: pos("testdata/defs.go:12:12 (offset 246)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "RoundTripper"}, Position: pos("testdata/defs.go:13:11 (offset 295)")},
+				&Ref{Def: Def{ImportPath: "net/http", PackageName: "http", Path: "Client"}, Position: pos("testdata/defs.go:20:13 (offset 392)")},
 			},
 		},
 		{
 			Filename: "testdata/vars.go",
 			Want: []*Ref{
-				&Ref{Def: Def{ImportPath: "net/http", Path: ""}, Position: pos("testdata/vars.go:3:8 (offset 21)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "Client"}, Position: pos("testdata/vars.go:6:14 (offset 74)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "RoundTripper"}, Position: pos("testdata/vars.go:8:12 (offset 124)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "Client Transport"}, Position: pos("testdata/vars.go:12:3 (offset 225)")},
-				&Ref{Def: Def{ImportPath: "net/http", Path: "Client"}, Position: pos("testdata/vars.go:11:12 (offset 194)")},
```

Comment:
the updated test cases don't really imply why this is important (later commits might though). From what I can tell this can just be computed from `ImportPath`, and if not, we should add a test for that.